generators/readme/package.json

```diff
@@ -16,15 +16,14 @@
     "@types/listr": "0.14.4",
     "@types/lodash": "4.14.194",
     "@types/mdast": "3.0.11",
-    "@types/mri": "1.1.1",
     "@types/node": "16.18.31",
     "@types/shelljs": "0.8.12",
     "@types/unist": "2.0.6",
     "@types/vfile": "4.0.0",
+    "cac": "^6.7.14",
```

Comment:
I was wondering if we should use a fixed version as we do with the rest of the dependencies 🤔